Source/Diagnostics/ReducedDiags/MultiReducedDiags.cpp

```diff
@@ -0,0 +1,101 @@
+#include "MultiReducedDiags.H"
+#include "AMReX_ParmParse.H"
+#include "AMReX_Utility.H"
+#include <fstream>
+
```

Comment:
We usually add
```c++
using namespace amex;
```
at the beginning of the `.cpp` files (**never** the `.H`!) files so that we don't have to put `amrex::` everywhere. You may do that if you want.

Reply:
Did that